shared_model/interfaces/base/signable.hpp

```diff
@@ -80,8 +80,10 @@ namespace shared_model {
        */
       bool operator==(const Model &rhs) const override {
         return this->hash() == rhs.hash()
-            and boost::equal(this->signatures(), rhs.signatures())
-            and this->createdTime() == rhs.createdTime();
```

Comment:
Is this check not necessary anymore?

Reply:
Yes, because create time is included into the payload.